Tactics/Reflect/AxEffects.lean

```diff
@@ -565,65 +565,123 @@ by default):
 - `eff.programProof`, and
 - `eff.stackAlignmentProof?` (if the field is not `none`)
 
-Return a list of fvar ids of the newly added hypotheses -/
+Return an `AxEffect` where these expression have been replaced by a reference
+to the newly created fvar -/
```

Comment:
"these expressions" refers to the list just above it of expressions that we added as hyps to the local context, i.e.
```
- one for every field in `eff.fields`
- `eff.nonEffectProof`,
- `eff.memoryEffectProof`,
- `eff.programProof`, and
- `eff.stackAlignmentProof?` (if the field is not `none`)
```

The point being that for the return value, all these fields will be `Expr.fvar ...` with the id of the hyp we just added to the local context.

This seemed a bit cleaner than just returning an array of FVarIDs, since the array completely lost all structure of which fvarId belong to which field

Reply:
I've reworded to
> Return an `AxEffect` where the expressions mentioned above have been replaced by
`Expr.fvar <fvarId>`, with `fvarId` the id of the corresponding hypothesis
that was just added to the local context